quiz/quiz.go

```diff
@@ -0,0 +1,168 @@
+/*
+	quiz defines a quiz and its questions and provides methods for starting and processing quizzes as well as reading in
+    a Quiz from file.
+*/
+package quiz
+
+import (
+	"bufio"
+	"encoding/csv"
+	"errors"
+	"fmt"
+	"io"
+	"math/rand"
+	"os"
+	"strings"
+	"time"
+)
+
+// A Quiz struct is a collection of up to 100 Questions, the name of the file containing the quiz, the number of
+// questions in the quiz and the number of questions the user got correct.
+type Quiz struct {
+	Questions        []Question
+	Filename         string
+	TotalQuestions   int
+	CorrectQuestions int
+}
+
+// NewQuiz creates a new Quiz struct based on the information in the file at the given file name
```

Comment:
Is there a reason to call this `NewQuiz` rather than just `New`.  Since the package is already called quiz we should name it `New` that way when you go to call it, it reads as: `quiz.New`.